### DISCUSSION TITLE
Nether Portal Spread compatibility

### DISCUSSION THREAD

**Discussion by BrandonPotter0** [DynamicTreesTeam/DynamicTrees#986] [Compatibility Addon Suggestions]
https://www.curseforge.com/minecraft/mc-mods/nether-portal-spread

It doesnt add any trees persay, but trees within the corruption area should get transformed into their warped nether versions.

**Comment by supermassimo** [comment]
This forum is for suggesting addons that extend Dynamic Trees with more trees from other mods.

Feature compatibility with an incompatible mod should be made as a suggestion issue instead.
